tools/generator/CodeGeneratorOptions.cs

```diff
@@ -104,14 +105,15 @@ public static CodeGeneratorOptions Parse (string[] args)
 					"SDK Platform {VERSION}/API level.",
 					v => opts.ApiLevel = v },
 				{ "lang-features=",
-					"For internal use. (Flags: interface-constants,default-interface-methods,nested-interface-types,nullable-reference-types,obsoleted-platform-attributes,restrict-to-attributes)",
+					"For internal use. (Flags: interface-constants,default-interface-methods,nested-interface-types,nullable-reference-types,obsoleted-platform-attributes,restrict-to-attributes,dont-fix-obsolete-overrides)",
```

Comment:
Could we call it `do-not-fix-obsolete-overrides` instead of `dont-…`?

Reply:
Yup, good call.